mne/io/tests/test_read_raw.py

```diff
@@ -99,3 +99,64 @@ def test_read_raw_multiple_dots(tmp_path):
     dst = tmp_path / "test.this.file.edf"
     copyfile(src, dst)
     read_raw(dst)
+
+
+@pytest.fixture(scope="module")
+def reader_excluded_from_read_raw() -> set[str]:
+    """Set of excluded readers from read_raw."""
+    return {
+        "read_raw_bti",
+        "read_raw_hitachi",
+        "read_raw_neuralynx",
+    }
```

Comment:
Any reason not to prefer the simpler:

```suggestion
reader_excluded_from_read_raw = {
    "read_raw_bti",
    "read_raw_hitachi",
    "read_raw_neuralynx",
}
```

To me this is simpler and fixtures should be preferred when you have to do some computation or something that will take some time etc.

Reply:
I prefer to use fixture to create any variables/resources (computationally expensive or not) that are required by a test. That's just my personal opinion, I committed your requested change here.